project/src/main/java/com/google/sps/data/PlacesFetcher.java

```diff
@@ -39,20 +46,22 @@
 
     /** In this radius around "LOCATION" places will be searched. */
     private static final int SEARCH_RADIUS = 5000;
-    // TODO(M1): check at least 10 results, and if less extend radius
 
     /** The entry point for a Google GEO API request. */
     private static final GeoApiContext CONTEXT = new GeoApiContext.Builder()
         .apiKey(System.getenv("API_KEY"))
         .build();
 
+    /** A mapping between cuisines and text search words. */
+    private static final Map<String, String[]> CUISINE_TO_SEARCH_WORDS_MAP = getCuisinesMap();
```

Comment:
Does it have to be an array? Prefer ImmutableList.

Reply:
I changed it to ArrayList. I didn't find a straight forward way to read the json values into an immutableList. I can keep struggling with this, but I thought keeping the whole map as an immutableMap with ArrayList as values is good enough. What do you think?

Reply:
Don't see the change on my end. Regarding your question, getCuisinesMap is the one returning the value, it can be intermediately be read as an array (or something else) and then converted to an ImmutableList by a simple conversion (ImmutableList.copyOf(/* array */).

Reply:
Yes, but then I have to iterate over the map that getCuisinesMap returns and do this conversion for every entry on it.
I suppose there is a way to do this while reading the json instead of building the whole map and then going back to modify each entry, but I didn't find something that worked.
Is this extra map iteration and lists copying worth it? Isn't it enogh for the map to be immutable?